model-governance/model_governance_graph.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>I'm not sure how this is passing CI.  You should provide the region based on the current sagemaker session...  you can instruct the customer in the markdown about options they have for changes here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2159/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>